yorkie/src/main/kotlin/dev/yorkie/document/operation/TreeStyleOperation.kt

```diff
@@ -29,16 +30,33 @@ internal data class TreeStyleOperation(
             YorkieLogger.e(TAG, "fail to execute, only Tree can execute edit")
             return emptyList()
         }
-        val changes = tree.style(fromPos to toPos, attributes.toMap(), executedAt)
 
-        return changes.map {
-            TreeStyleOpInfo(
-                it.from,
-                it.to,
-                it.fromPath,
-                it.attributes.orEmpty(),
-                root.createPath(parentCreatedAt),
-            )
+        return when {
+            attributes?.isNotEmpty() == true -> {
+                tree.style(fromPos to toPos, attributes, executedAt).map {
+                    TreeStyleOpInfo(
+                        it.from,
+                        it.to,
+                        it.fromPath,
+                        it.attributes.orEmpty(),
+                        path = root.createPath(parentCreatedAt),
+                    )
+                }
+            }
+
+            attributesToRemove?.isNotEmpty() == true -> {
+                tree.removeStyle(fromPos to toPos, attributesToRemove, executedAt).map {
+                    TreeStyleOpInfo(
+                        it.from,
+                        it.to,
+                        it.fromPath,
+                        attributesToRemove = it.attributesToRemove.orEmpty(),
+                        path = root.createPath(parentCreatedAt),
+                    )
+                }
+            }
+
+            else -> emptyList()
```

Comment:
is it assured that only one of `attributes` or `attributesToRemove` will be set?

Reply:
In the current interface, `style()` can only set `attributes`, and `removeStyle()` can only set `attributesToRemove`. So I think it is logically assured that only one of them will be set.